crates/pera-framework/Makefile

```diff
@@ -0,0 +1,9 @@
+build:
+	cargo build --release
+test:
+	cd .. && pwd && cargo run --bin pera -- move test proof_tests --path ./pera-framework/packages/pera-system/tests
+test-bin:
+	./../../target/debug/pera move test proof_tests --path ./packages/pera-system/tests
+snapshot:
+	cd .. && UPDATE=1 cargo test -p pera-framework --test build-system-packages && cargo run --bin pera-framework-snapshot
```

Comment:
Why run the tests when creating a snapshot?

Reply:
In the new version it's needed to re-build the MOVE code AFAIK